pyarrow-stubs/__lib_pxi/ipc.pyi

```diff
@@ -154,6 +154,8 @@ class _RecordBatchFileReader(_Weakrefable):
     def __enter__(self) -> Self: ...
     def __exit__(self, exc_type, exc_val, exc_tb): ...
     @property
+    def schema(self) -> Schema: ...
```

Comment:
mentioned in [the docs](https://arrow.apache.org/docs/python/generated/pyarrow.ipc.RecordBatchFileReader.html#pyarrow.ipc.RecordBatchFileReader.schema) (though it's not explicit about the type) / [source link](https://github.com/apache/arrow/blob/6a2e19a852b367c72d7b12da4d104456491ed8b7/python/pyarrow/ipc.pxi#L1061-L1062) (with type)